pkg/network/node/networkpolicy.go

```diff
@@ -64,7 +68,10 @@ type npNamespace struct {
 type npPolicy struct {
 	policy            networking.NetworkPolicy
 	watchesNamespaces bool
-	watchesPods       bool
+	// not used in 3.11
+	// added in https://github.com/openshift/sdn/commit/3c35081bda7f4e0be1ae0f5e85f3ad3b3e6d1224
```

Comment:
yeah, this depends on other changes that weren't backported to 3.11. I'd drop `networkpolicy: consolidate OVS transactions` and `networkpolicy: do policy recalculation in the bfr thread to avoid blocking`. Basically, take the patches involved in getting rid of `np.pods`, but not the rest, at least for now.

Reply:
will do